docs/README.md

```diff
@@ -0,0 +1,42 @@
+# The SpatialOS GDK for Unreal (pre-alpha) documentation on GitHub
```

Comment:
So I think with the addition of the website docs, we shouldn't have this readme anymore? As in maybe this readme should just link back to the site.

Reply:
Because most of those docs are going to be unreadable.

Reply:
(1) For the user:
The GitHub docs are completely unreadable but it's not clear that there is an alternative or why. You look at the docs and they just look completely rubbish. The information on the repo /README and the docs/README makes it clearer how to proceed. The README docs display by default when you navigate to the directory so it makes things really clear.

(2) For anyone updating the docs:
We have to update via the GitHub repo, so we need to have an understanding of how the docs are structured -  it's very confusing for me (and I made the docs!) as there's a directory structure but I can't see how it all works - the ToC in the docs/README helps us without having to go to the website and try to infer how it works.

Reply:
The existence of a toc in the `docs/README` implies that you can navigate the github docs, which is what I would potentially do as a user since I skim large amounts of information. My eyes would see a toc and start clicking, not reading what's at the top of the file. I think going to the website or `toc.md` is an okay compromise for clarity in the repo. Having two lists of the same information is going to make them get out of sync at certain points.

Reply:
Just thought - I could make the ToC a comment...